libs/src/services/ethContracts/serviceTypeManagerClient.ts

```diff
@@ -0,0 +1,112 @@
+import { Utils } from '../../utils'
+import { GovernedContractClient } from '../contracts/GovernedContractClient'
+
+export class ServiceTypeManagerClient extends GovernedContractClient {
+  /**
+   *
+   * @param serviceType Type of service to set the version, either `discovery-node` or `content-node`
+   * @param serviceVersion Version string to set on chain
+   * @param privateKey Optional privateKey to pass along to web3Manager sendTransaction
+   * @param dryRun Optional parameter to return the generated parameters without sending tx
+   * @returns comma-separated String of serviceType and serviceVersion if dryRun; else response from web3Manager.sendTransaction
+   */
+  async setServiceVersion(
+    serviceType: string,
+    serviceVersion: string,
+    privateKey: string | null = null,
+    dryRun = false
+  ) {
+    const method = await this.getGovernedMethod(
+      'setServiceVersion',
+      Utils.utf8ToHex(serviceType),
+      Utils.utf8ToHex(serviceVersion)
+    )
+
+    if (dryRun) {
+      return `${Utils.utf8ToHex(serviceType)},${Utils.utf8ToHex(
+        serviceVersion
+      )}`
+    }
+
+    return await this.web3Manager.sendTransaction(
+      method,
+      await this.governanceClient.getAddress(),
+      privateKey
+    )
+  }
+
+  async addServiceType(
+    serviceType: string,
+    serviceTypeMin: string,
+    serviceTypeMax: string,
```

Comment:
These should be type BN - ref: https://github.com/AudiusProject/audius-protocol/blob/82bac7496cd9a2ed79c77f064d3c54a5efe9c6f8/libs/eth-contracts/ABIs/ServiceTypeManager.json#L115-L139